evap/staff/tests/test_importers.py

```diff
@@ -149,9 +153,28 @@ def test_disallow_non_string_types(self):
             set(errors[ImporterError.SCHEMA])
         )
 
+    def test_user_success_messages(self):
+        user_list, success_messages, warnings, errors = UserImporter.process(self.valid_excel_content, test_run=True)
+        self.assertIn('The import run will create 2 users:<br />Lucilia Manilium (lucilia.manilium@institution.example.com)<br />Bastius Quid (bastius.quid@external.example.com)', success_messages)
```

Comment:
It is a bit odd that the full html is here, although that is not really what you want to test right now, maybe you could check for only the message and then check the two names in separate asserts. I am not even sure whether the check for the names is appropriate here